### DISCUSSION TITLE
ValueError: Non-consecutive added token '<extra_id_99>' found when loading 'CogVideoXPipeline'

### DISCUSSION THREAD

**Discussion by daeunni** [huggingface/diffusers#9246] [Q&A]
Hi, I found the below error when I tried to load `CogVideoXPipeline` as follows: 
`pipe = CogVideoXPipeline.from_pretrained("THUDM/CogVideoX-2b").to("cuda")`

--> `ValueError: Non-consecutive added token '<extra_id_99>' found. Should have index 32100 but has index 32000 in saved vocabulary.`

How can I address it? My diffusers version is v0.30.0. Thanks!

**Comment by tolgacangoz** [comment]
Hi @daeunni. Could you try to upgrade your `transformers`?
Also, if you want to use [`CogVideoX`'s latest improvements](https://huggingface.co/docs/diffusers/main/en/api/pipelines/cogvideox) install `diffusers` from the source.
